docs/source/contributing/setup.rst

```diff
@@ -93,21 +99,33 @@ happen.
       python3 -m pip install -r dev-requirements.txt
       python3 -m pip install -r requirements.txt
 
-5. Install the development version of JupyterHub. This lets you edit
+5. Setup a database.
+
+   The default database is ``sqlite`` so if you are just trying
+   to get up and running quickly for local development remember to install
+   the native package for your distribution, for example on Ubuntu:
+
+   .. code:: bash
+
+      sudo apt-get install sqlite3
```

Comment:
Should I just remove this part starting with the "for example..."? The "install the native package for your distribution" is generic enough I think.

Reply:
Just as an FYI this is what I have installed now for lubuntu 18.04 (and I think these were part of the base image I used to create this VM):

```
$ dpkg -l | grep sqlite
ii  libsqlite0                             2.8.17-14fakesync1                              amd64        SQLite 2 shared library
ii  libsqlite3-0:amd64                     3.22.0-1ubuntu0.2                               amd64        SQLite 3 shared library
ii  libsqlite3-dev:amd64                   3.22.0-1ubuntu0.2                               amd64        SQLite 3 development files
```